.lando.yml

```diff
@@ -0,0 +1,27 @@
+name: amphtml
+
+services:
+  appserver:
+    type: node:10
+    build_as_root:
+      - curl --compressed -o- -L https://yarnpkg.com/install.sh | bash
+    build:
+      - yarn global add gulp-cli
+      - yarn
+    run_as_root:
+      - echo "127.0.0.1 amphtml.lndo.site" >> /etc/hosts
+    command: cd /app && yarn gulp serve --port=80 --host=0.0.0.0
+
+tooling:
+  yarn:
+    service: appserver
+  gulp:
+    service: appserver
+    cmd: "yarn gulp"
+  watch:
+    service: appserver
+    cmd: "yarn gulp watch"
```

Comment:
The sum total of all our `gulp` commands used during development can be found by running `gulp help`. Other commands that might be useful to add here: `check-types`, `dist`, `unit`, `integration`, and `e2e`.